scripts/run.sh

```diff
@@ -2,6 +2,12 @@
 set -e
 
 # Setup
+if command -v docker-compose &> /dev/null
+then
+  dccmd='docker-compose'
+else
+  dccmd='docker compose'
+fi
```

Comment:
To follow our bash coding standards:
```suggestion
then
    dccmd='docker-compose'
else
    dccmd='docker compose'
fi
```